docs/app-integration-development/09-Classic-User-OAuth-Auth-Code-Grant.md

```diff
@@ -87,7 +87,7 @@ The access token will be included in a JSON response. You may also want to take
 
 Note however, that our access tokens do expire after a defined period of time -- so you may want to make sure that you implement OAuth refresh to prevent users needing to re-authorize your app. See more information about token expiries at the bottom of this page.
 
-For additional information about the user, account, and PagerDuty service region where your app is now authorized, you can look at [cracking open our PagerDuty-signed ID token](../../docs/app-integration-development/11-OAuth-2-Id-token.md). For example, the `aud` field will help your integration with data residency and processing guarantees if you have customers located in Europe.
+For additional information about the user, account, and PagerDuty service region where your app is now authorized, you can look at [cracking open our PagerDuty-signed ID token](../../docs/app-integration-development/11-Classic-User-OAuth-2-Id-token.md). For example, the `aud` field will help your integration with data residency and processing guarantees if you have customers located in Europe.
```

Comment:
i think there's an extra `-2` in the link here 🤔

Reply:
oops, ty!